boa/src/builtins/object/mod.rs

```diff
@@ -54,7 +54,7 @@ impl BuiltIn for Object {
         .name(Self::NAME)
         .length(Self::LENGTH)
         .inherit(JsValue::null())
-        .method(Self::has_own_property, "hasOwnProperty", 0)
+        .method(Self::has_own_property, "hasOwnProperty", 1)
```

Comment:
With this change, `test/built-ins/Object/prototype/hasOwnProperty` conformance is now 100%.

Reply:
I suspect similar bug to be present elsewhere, e.g. `propertyIsEnumerable` below.